js/main.js

```diff
@@ -22,4 +32,92 @@ $form.addEventListener('submit', function (event) {
   data.nextEntryId++;
 
   $form.reset();
+
+  const $newDOM = renderEntry(obj);
+
+  $ul.prepend($newDOM);
+
+  viewSwap('entries');
+
+  if (data.entries.length <= 0) {
+    toggleNoEntrires();
+  }
+});
+
+// assigns DOM tree
+
+function renderEntry(entry) {
+  // <li class='row'>
+  // <div class='column-half'>
+  //   <img src='$photoURL' alt="image"/>
+  // </div>
+  // <div class='column-half'>
+  //   <h4>A newer image</h4>
+  //   <p>A very nice lake</p>
+  // </div>
+  // </li>
+
+  const $li = document.createElement('li');
+  $li.setAttribute('class', 'row');
+
+  const $divHalfOne = document.createElement('div');
+  $divHalfOne.setAttribute('class', 'column-half');
+
+  const $img = document.createElement('img');
+  $img.setAttribute('src', entry.url);
+
+  const $divHalfTwo = document.createElement('div');
+  $divHalfTwo.setAttribute('class', 'column-half');
+
+  const $h4 = document.createElement('h4');
+  $h4.textContent += entry.title;
+
+  const $p = document.createElement('p');
+  $p.textContent += entry.notes;
+
+  $li.append($divHalfOne, $divHalfTwo);
+  $divHalfOne.append($img);
+  $divHalfTwo.append($h4, $p);
+
+  return $li;
+}
+
+// pushes data into list
+
+document.addEventListener('DOMContentLoaded', function (event) {
+  for (let i = 0; i < data.entries.length; i++) {
+    const $finalDOM = renderEntry(data.entries[i]);
+    $ul.appendChild($finalDOM);
+  }
+
+  viewSwap(data.view);
+
+  if (data.entries.length > 0) {
+    toggleNoEntrires();
+  }
+});
+
+function toggleNoEntrires() {
+  if (data.entries.length <= 0) {
+    $noEntries.setAttribute('class', 'no-entries p1');
+  }
```

Comment:
You need an `else` that does the opposite so that you hide it if we do have entries